client/lib/performance-tracking/components/performance-tracker-stop.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { isSingleUserSite, isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import isSiteStore from 'state/selectors/is-site-store';
+import { stopNavigation } from '../api';
+
+const PerformanceTrackerStop = ( { single, at, jetpack, store } ) => {
```

Comment:
Do we need to track whether we've already called `stopNavigation` for this component instance to prevent multiple calls (in the case of a re-render for example)?

Reply:
`LUX` takes care of that for us. We can call `LUX.send()` many times and it only will send the first one. The rest will be ignored.

But I agree we'll need to improve our start/stop detection if/when we see the value of RUM stats, to have all the things you suggested as automatic stop detection, avoid double calls, etc.